js/components/graph/Graph.js

```diff
@@ -748,12 +786,14 @@ export default class Graph extends React.Component {
   }
 }
 
+export {ZOOM_INCREMENT, KEYBOARD_PANNING_INCREMENT}
+
 /** Helper function for parsing hybrid node's text
  *
  * @param {string} s
  * @returns {Array}
  */
-function parseAnd(s) {
+ function parseAnd(s) {
```

Comment:
Revert this change and all changes in this file below this line (including the removal of a new line to end the file).